int8 visionStateCommand

string personName
